### DISCUSSION TITLE
Error message when husk chimera get decapitated while "eating"

### DISCUSSION THREAD

**Discussion by kageokami** [FakeFishGames/Barotrauma#14810] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When husk chimera get decapitated while eating, the debug consle throws a bunch of
[Vanilla] Character "Husk_chimera" failed to eat a target (No head limb defined)
![20241011190237_11](https://github.com/user-attachments/assets/362ed6eb-7ee4-4e26-b051-e790d4f9257d)
It is rare to happen without cheats, but indeed.

### Reproduction steps

The consistent way to reproduce this:
1. godmod
2. give "Boarding Axe"
3. spawn Husk_chimera cursor
4. attack
![20241011203327_1](https://github.com/user-attachments/assets/22279b8e-134e-432c-bfe3-c79e8308e8b2)
Another way is attack chimera when it eating someone else, which is how exactly I get this error at first time.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
[Vanilla] Character "Husk_chimera" failed to eat a target (No head limb defined)
No crashes and not affect the game play
```

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next Unstable build. Closing discussion.
